flink-dstl/flink-dstl-dfs/src/main/java/org/apache/flink/changelog/fs/StateChangeFsUploader.java

```diff
@@ -54,6 +55,8 @@
 public class StateChangeFsUploader implements StateChangeUploader {
     private static final Logger LOG = LoggerFactory.getLogger(StateChangeFsUploader.class);
 
+    private static final String DSTL_SUB_PATH = "%s/dstl";
+
```

Comment:
Could you explain the purpose of the `/dstl` suffix here?
Isn't this `DSTL_SUB_PATH` already appended to a path where changelog files reside?

Reply:
This purpose is described in the last paragraph of the [FLINK-28172](https://issues.apache.org/jira/browse/FLINK-28172).

> Going a step further, we can even create two levels of subdirectories under the dstl.dfs.base-path directory, like base-path/{jobId}/dstl . This way, if the user configures the same dstl.dfs.base-path as state.checkpoints.dir, all files needed for job recovery will be in the same directory and well organized.

WDYT ?